CMakeLists.txt

```diff
@@ -324,7 +324,8 @@ Enable this if the module uses typeid or dynamic_cast. You will probably need to
         sdk/include/ddk
         sdk/include/ndk
         sdk/include/reactos
-        sdk/include/reactos/libs)
+        sdk/include/reactos/libs
+        sdk/lib/pseh/include)
```

Comment:
Why did you add it globally instead of `public` on the `pseh` library?
Then only projects linking to `pseh` will get the include

Reply:
That has already been a problem in the past, see my previous fix to usetup: To use PSEH in setuplib, setuplib needs to link to pseh library, which in turn will cause usetup to link to link to pseh, but with a messed up link order, where pseh is linked after ntdll, resulting in link-errors. The fix was to remove the link to pseh and add the include dir instead.
Additionally excpt.h will be included in other modules as well, requiring the include path. It just makes things easier.

Reply:
ah, that is very bad of gcc :(